Headlines/src/Controllers/NewsTableViewController+FetchNews.swift

```diff
@@ -149,7 +149,21 @@ extension NewsTableViewController {
             break
         }
 
-        newsDataSource.fetchNews(page: lastPage, success: success, fail: fail)
+        let handler: ((Result <[News], Error>) -> Void) = { [weak self] result in
+            switch result {
+            case .success(let news):
+                self?.updateNews(news, mode)
+
+            case .failure(_):
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/cafeb7b8636cbafd0216eaa2489bb108aa52959c/Headlines/src/Controllers/NewsTableViewController+FetchNews.swift#L157"></span>Arguments can be omitted when matching enums with associated types if they are not used.<br />
<code>empty_enum_arguments</code> <code>NewsTableViewController+FetchNews.swift:157</code></td>
    </tr>
  </tbody>
</table>